qt_dotgraph/src/qt_dotgraph/dot_to_qt.py

```diff
@@ -226,6 +226,10 @@ def addEdgeItem(self, edge, nodes, edges, highlight_level, same_label_siblings=F
                              edge_color=color,
                              style=style)
 
+        edge_name = source_node.strip('"\n"') + '_TO_' + destination_node.strip('"\n"')
+        if label is not None:
+          edge_name = edge_name + '_' + label
+
```

Comment:
Please move these lines above line 243  since `edge_name` is not used until then.